recipes/libint2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libint" %}
+{% set version = "2.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/evaleev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 253fa3301e6742f5aac9e96dbe2a5707fc73abd6f4a214a8d44f03e113e69f00
+
+build:
+  number: 0
+  binary_relocation: true
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - boost
```

Comment:
Should this be `boost-cpp`, I know it is confusing, but `boost` actually includes the boost::python tool, which brings in Python, that you might not need.

Reply:
@scopatz thanks for the advice, I did not know about this.